### DISCUSSION TITLE
What 🎥 starring Johnny Depp was recently shown at this year's Cannes Film Festival?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#869] [Galaxy Trivia] [answered]
**Bonus:** What is the character he plays in this movie?

**Comment by peterbaccaro** [comment, marked as answer]
Johnny Depp's new film **Jeanne du Barry** was recently shown at the Cannes Film Festival on Tuesday (16 May). The French-language film directed by Maïwenn was controversially chosen to open the festival this year.

Johnny Depp stars as **King Louis XV** in "Jeanne du Barry".
